skops/_min_dependencies.py

```diff
@@ -16,7 +16,8 @@
     "pytest": (PYTEST_MIN_VERSION, "tests", None),
     "pytest-cov": ("2.9.0", "tests", None),
     "flake8": ("3.8.2", "tests", None),
-    "mypy": ("0.770", "tests", None),
+    "mypy": ("0.971", "tests", None),
```

Comment:
mypy isn't really a test dependency. It probably makes more sense to have it explicitly installed in CI the same as black and isort.